modules/generating-connectivity-difference-reports-with-the-roxctl-netpol-connectivity-diff-command.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * operating/build-time-network-policy-tools.adoc
+
+:_content-type: PROCEDURE
+[id="generating-connectivity-difference-reports-with-the-roxctl-netpol-connectivity-diff-command_{context}"]
+= Generating connectivity difference reports with the roxctl netpol connectivity diff command
+
+
+To produce a connectivity difference report, the `roxctl netpol connectivity diff` command requires two folders, `dir1` and `dir2`, each containing Kubernetes manifests, including network policies.
+
```

Comment:
It seems fine here but be careful about extra blank lines--sometimes they can cause issues with HTML output.